src/main/java/com/klachkova/locationsystem/services/LocationService.java

```diff
@@ -106,23 +115,25 @@ private void validateLocation(Location location) {
      * @throws NotFoundException if no location with the given address is found
      */
     public Location findByAddress(String address) {
+
         return locationRepository.findByAddress(address)
-                .orElseThrow(() -> new NotFoundException("Location with address " + address + " not found"));
+            .orElseThrow(() -> new NotFoundException("Location with address " + address + " not found"));
     }
 
     /**
-     * Retrieves a location by its ID.
-     * <p>
-     * Throws an exception if no location with the given ID is found.
-     * </p>
+     * Retrieves a Location by its ID.
+     * First checks the Redis cache for the location.
+     * If the location is not found in the cache, it retrieves
+     * the location from the database and caches it for future access.
```

Comment:
Please check if its correct place for this comment, probably its related to `etAvailableLocations`?

Reply:
I initially implemented the cache for these methods too, then I deleted the changes to the methods and forgot about the documentation.
Edited